Dockerfile

```diff
@@ -15,11 +15,11 @@ ARG ARCH=x86_64
 ARG BUILD_MACHINE_ARCH=x86_64
 ARG BUILDARCH=amd64
 ARG TRIPLET=${ARCH}-linux-gnu
-ARG GIT_SHA="unknown"
-ARG BUN_DOWNLOAD_URL_BASE="https://pub-5e11e972747a44bf9aaf9394f185a982.r2.dev/releases/latest"
+ARG GIT_SHA=""
```

Comment:
note, bun zig code already handles git sha being 0, so this is best for unknown hash